rpc/README.md

```diff
@@ -4842,138 +4841,6 @@ socket.send(`{"id": 2, "jsonrpc": "2.0", "method": "unsubscribe", "params": ["0x
     * `topic`: `string`
 * result: `string`
 
-Subscribes to a topic.
```

Comment:
It looks like the documentation for the rpc `subscribe` method is not being generated correctly.

Reply:
yes, let me reconsider how to generate API docs.